unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/FriendsController/IFriendsApiBridge.cs

```diff
@@ -22,8 +22,8 @@ public interface IFriendsApiBridge
         void RejectFriendship(string userId);
         UniTask<RejectFriendshipPayload> RejectFriendshipAsync(string friendRequestId);
         void RemoveFriend(string userId);
-        void GetFriends(int limit, int skip);
-        void GetFriends(string usernameOrId, int limit);
+        UniTask<AddFriendsPayload> GetFriendsAsync(int limit, int skip, CancellationToken ct);
```

Comment:
its a common practice to set the cancellation token with a default value: `CancellationToken cancellationToken = default`

Reply:
Done!